scripts/profiler/rtt_nordic_profiler_host.py

```diff
@@ -45,6 +45,13 @@ def __init__(self, config=RttNordicConfig, finish_event=None,
         self.logger_console.setFormatter(self.log_format)
         self.logger.addHandler(self.logger_console)
 
+        self.rtt_up_channels = {
+            'info' : None,
+            'data' : None,
+        }
+        self.rtt_down_channels = {
+            'command' : None,
```

Comment:
same comment regarding spaces.
```suggestion
            'info': None,
            'data': None,
        }
        self.rtt_down_channels = {
            'command': None,
```

Reply:
fixed